Counties/Florida/Bay County/Scraper/tests/test_ScraperUtils.py

```diff
@@ -54,6 +57,38 @@ def test_parse_plea_type_not_guilty(self):
         assert ScraperUtils.parse_plea_type(plea2) == 'Not Guilty'
         assert ScraperUtils.parse_plea_type(plea3) == 'Not Guilty'
 
+    def test_parse_name(self):
+        name1 = "DOE, JANE EMILY"
+        name2 = "DOE, JOHN"
+        assert ScraperUtils.parse_name(name1) == ('JANE', 'EMILY', 'DOE')
+        assert ScraperUtils.parse_name(name2) == ('JOHN', None, 'DOE')
```

Comment:
Optional if you want to change all these, but [pytest supports parameterization](https://docs.pytest.org/en/stable/parametrize.html), if you weren't aware.